src/languages/es.js

```diff
@@ -30,6 +30,8 @@ export default {
         lastName: 'Apellido',
         phone: 'teléfono',
         phoneNumber: 'Número de teléfono',
+        phonePlaceholder: '(prefijo) + (número)',
+        errorMessageInvalidPhone: 'Si reside fuera de Estados Unidos, por favor incluye el prefijo internacional. P. ej. +447782339811',
```

Comment:
Here you have a dot at the end. Did you confirm this with the team?

Reply:
No I didn't as we were already using them.